### DISCUSSION TITLE
Open Action is missing the working folder

### DISCUSSION THREAD

**Discussion by Tragen** [elgatosf/elgato#24] [Ideas]
On Windows when I open a app, the Windows Temp folder is used as working folder so e.g. my Terminal starts with a strange folder.
Other apps are also having problems to start if the require the working folder to be set the same as the exe like Explorer is doing.

**Comment by ChekTek** [comment]
I'm not sure if I fully understand your question, but I think you can work around this using command line arguments in the open action to provide a working directory to your application. 
```
"C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe" --profile-directory="Profile 1" --allow-file-access-from-files
```
You may also find some success creating a shortcut for the application you are opening.

**Comment by Tragen** [comment]
No, that's not what I mean.

If you look at the properties of a shortcut in Windows, you see the "Start in" and that is missing here.

![image](https://github.com/elgatosf/elgato/assets/1700241/ba9858c6-3a04-4647-b6ff-5bc35537275b)

**Reply by ChekTek** [reply]
I see what you mean I will take this feedback to the team. I believe you can create a shortcut and reference that from the Stream Deck open action, to get the same result.

**Comment by Tragen** [comment]
How is it solved? Is there a new update coming?

**Reply by ChekTek** [reply]
I've created a ticket for this internally, so the Stream Deck core team can prioritize as they see fit.
